### PR TITLE
ENH: coreg UI

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This small PR extracted from #10224 
* renames the MRI groupbox from "Transform" to "HEAD <> MRI Transform"
* renames the info groupbox from "Digitization" to "Info source with digitization"
* removes `project_eeg` from the view options ~(and only the view options, it's still possible to enable it from command line for example)~

plus minor edits

### CODE REVIEW

**Comment by larsoner** [comment]
Let's get rid of it from the command line, too. YAGNI and we get more `-` lines that way

**Comment by GuillaumeFavelier** [comment]
> Let's get rid of it from the command line, too

Perfect! I was wondering actually

**Comment by GuillaumeFavelier** [comment]
I ended up removing a lot of lines in the end.

**Comment by GuillaumeFavelier** [comment]
What is the best way to deal with `mne/tests/test_docstring_parameters.py` with so many changes to the doc of `coregistration()`?

I do not have this behaviour with `CoregistrationUI`

**Comment by larsoner** [comment]
The parameters in the signature should match the set in the docstring. If some are deprecated, you just change their description like:
```
project_eeg : bool
    Deprecated. Use :func:`mne.viz.plot_alignment` to see projected EEG electrodes.
```

**Comment by GuillaumeFavelier** [comment]
My diff is really boring:

```
tabbed : bool
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
split : bool
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
scrollable : bool
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
head_inside : bool
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
guess_mri_subject : bool
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
scale : float
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
project_eeg : bool
	Deprecated. Use :func:`mne.viz.plot_alignment` to see projected EEG electrodes.
advanced_rendering : bool
	Deprecated. This parameter is not supported by the pyvistaqt 3d backend.
```

**Comment by hoechenberger** [comment]
> My diff is really boring:

I liked this one 😅

**Comment by GuillaumeFavelier** [comment]
I decided to remove only `project_eeg` in here because I do not want to delay this further (it was supposed to be a small one :sweat_smile: ). We can always remove the rest in another PR.

**Comment by GuillaumeFavelier** [comment]
I do not understand why this fails on `linux / conda / py3.9`. I'll try locally on a new env.

**Comment by hoechenberger** [comment]
@GuillaumeFavelier I think I've seen this issue before locally after unsetting the `SUBJECTS_DIR` environment variable und passing `subjects_dir=None`

**Comment by GuillaumeFavelier** [comment]
Do you have an idea on how to fix this? It does not make sense to me because `CoregistrationUI` *should* send the  `DeprecationWarning` way before `_setup_bem()` but clearly there is something I missed.

**Comment by GuillaumeFavelier** [comment]
And I just tried on a clean conda env, I cannot reproduce.

**Comment by GuillaumeFavelier** [comment]
Explaining it to you made me realize the issue @hoechenberger :sweat_smile: 

Indeed it's only a `DeprecationWarning` so it does not interrupt the execution and it does continue into `_setup_bem()`.

I'll push something to fix that :+1:

**Comment by hoechenberger** [comment]
Perfect! @GuillaumeFavelier I'll be home in 30 min and can test then. Also happy to make a screen share session if you want! Just ping me

File: mne/gui/_coreg.py [hunk 1]
```diff
@@ -126,10 +122,10 @@ def __init__(self, info_file, subject=None, subjects_dir=None,
                  fiducials='auto', head_resolution=None,
                  head_opacity=None, hpi_coils=None,
                  head_shape_points=None, eeg_channels=None, orient_glyphs=None,
-                 scale_by_distance=None, project_eeg=None, mark_inside=None,
+                 scale_by_distance=None, mark_inside=None,
```
Comment (larsoner) [inline mne/gui/_coreg.py:125]: Or maybe earlier in the list? Not sure
```suggestion
                 scale_by_distance=None, *, mark_inside=None,
```

**Review by hoechenberger — APPROVED** [review]

**Comment by hoechenberger** [comment]
Thanks @GuillaumeFavelier!

**Pull request merged**